components/expenses/ExpenseMoreActionsButton.js

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Check } from '@styled-icons/feather/Check';
+import { ChevronDown } from '@styled-icons/feather/ChevronDown/ChevronDown';
+import { Download as IconDownload } from '@styled-icons/feather/Download';
+import { Edit as IconEdit } from '@styled-icons/feather/Edit';
+import { Flag as FlagIcon } from '@styled-icons/feather/Flag';
+import { Link as IconLink } from '@styled-icons/feather/Link';
+import { useRouter } from 'next/router';
+import { FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { margin } from 'styled-system';
+
+import expenseTypes from '../../lib/constants/expenseTypes';
+import useClipboard from '../../lib/hooks/useClipboard';
+import { getCollectivePageRoute } from '../../lib/url-helpers';
+
+import { Flex } from '../Grid';
+import PopupMenu from '../PopupMenu';
+import StyledButton from '../StyledButton';
+
+import ExpenseInvoiceDownloadHelper from './ExpenseInvoiceDownloadHelper';
+import MarkExpenseAsIncompleteModal from './MarkExpenseAsIncompleteModal';
+
+const Action = styled.button`
+  ${margin}
+  padding: 16px;
+  cursor: pointer;
+  line-height: 16px;
+  font-size: 14px;
+  font-weight: 500;
+  border: none;
+  background: transparent;
+  outline: none;
+  text-align: inherit;
+
+  color: ${props => props.theme.colors[props.color]?.[500] || props.color || props.theme.colors.black[900]};
+
+  :hover {
+    color: ${props => props.theme.colors[props.color]?.[300] || props.color || props.theme.colors.black[700]};
+  }
+
+  &[disabled] {
+    color: ${props => props.theme.colors[props.color]?.[200] || props.color || props.theme.colors.black[600]};
+  }
```

Comment:
Seems quite complex, in this kind of situations I think it's better to rely on `styled-system`'s `color` helper:

```jsx
import { color } from 'styled-system';

const MyDiv = styled.div`
  ${color}
`

return <MyDiv color="black.600 />;
```

But here I don't see us using the prop at all, is it maybe a leftover? In which case we can simplify

```suggestion
  color: ${props => props.theme.colors.black[900]};

  :hover {
    color: ${props => props.theme.colors.black[700]};
  }

  &[disabled] {
    color: ${props => props.theme.colors.black[600]};
  }
```